lib/kafka-producer.js

```diff
@@ -70,41 +70,40 @@ Producer.prototype.getProducer = Promise.method(function (opts, topicOptions) {
  * @private
  */
 Producer.prototype._serializeType = function (topicName, isKey, data) {
-
   let schema = null;
   let subject = null;
 
```

Comment:
* removing `this.sr.keySchemas` and `this.sr.valueScheams` in favor of a single schema container `this.sr.schemas`.
* removing schemaType postfix ( -key, -value )